### ISSUE TITLE
Comet tower incorrect price

### ISSUE THREAD

**Issue by decoy11** [space-jam-/vampirism#263]
I'm not sure what the accurate price is but I feel like there is a digit missing making it cost 1/10th of the wood and maybe the gold price is being ignored.

**Comment by decoy11** [comment]
Okay this bug should be pushed to being fixed asap. The game just ends once humans can access comet towers. The power creep is too much for any vampire to do anything.

**Comment by MaxwellLevenson** [comment]
Was this the Vampire Spire Tower? I think the Comet Tower had the correct price but a fix was just pushed to the cost of the Vampire Spire Tower where it was incorrectly 15k wood instead of 75k wood and 30 gold

**Comment by decoy11** [comment]
Yes Vampire tower is actually the Comet tower. I am going to close this issue because it is now fixed

**Comment by decoy11** [comment]
No wait current version 0.15b as of 9/25/2015 still has the wrong price.

dota2 client has this in the npc_abilities_custom.txt

start line: 7813  //================================================================================= 
  // Build Cripple Tower
  //=================================================================================
  "build_tower_cripple_t1"
  {
    "AbilityBehavior"           "DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_IMMEDIATE"
    "BaseClass"                 "ability_datadriven"
    "AbilityTextureName"        "house_t1"
    "AbilityCastAnimation"      "ACT_DOTA_DISABLED"
    // BuildingHelper info
    "Building"                  "1"
    "BuildingSize"              "2"
    "BuildTime"                 "30"
    "AbilityCastRange"          "200"
    "UpdateHealth"              "1"
    "Scale"                     "1"
    "MaxScale"                  "0.7"
    "CasterCanControl"          "1"

    "AbilityCooldown"           "0"
    "LumberCost"                "15000"
    
Github also has the incorrect values in npc_abilities_custom.txt

//================================================================================= 
  // Build Cripple Tower
  //=================================================================================
  "build_tower_cripple_t1"
  {
    "AbilityBehavior"           "DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_IMMEDIATE"
    "BaseClass"                 "ability_datadriven"
    "AbilityTextureName"        "house_t1"
    "AbilityCastAnimation"      "ACT_DOTA_DISABLED"
    // BuildingHelper info
    "Building"                  "1"
    "BuildingSize"              "2"
    "BuildTime"                 "30"
    "AbilityCastRange"          "200"
    "UpdateHealth"              "1"
    "Scale"                     "1"
    "MaxScale"                  "0.7"
    "CasterCanControl"          "1"

    "AbilityCooldown"           "0"
    "LumberCost"                "75000"
    "GoldCost"                  "30"

The correct values should be 125000 lumber and 40 gold
